control/grpc.py

```diff
@@ -73,40 +73,8 @@ def __init__(self, config: GatewayConfig, gateway_state: GatewayStateHandler, rp
         """Constructor"""
         self.gw_logger_object = GatewayLogger(config)
         self.logger = self.gw_logger_object.logger
+        config.display_environment_info(self.logger)
```

Comment:
@gbregman, looks like this display_environment_info() will be called after control/__main__.py:40

I would remove this call and `self.env_shown = False` flag.

Reply:
@baum I've done the same with the config dump. In my environment it's not needed but I wasn't sure about other configuration. So, I decided to use it. The cost is almost zero. It's one extra if  executed only once on startup.